src/components/ContactCard/ContactFieldInput.jsx

```diff
@@ -3,10 +3,7 @@ import PropTypes from 'prop-types'
 import { Field } from 'react-final-form'
 
 import HasValueCondition from '../Form/HasValueCondition'
-import { AdaptedInput, AdaptedTextarea } from '../Form/fields'
-
-const getInputComponent = inputType =>
-  inputType === 'textarea' ? AdaptedTextarea : AdaptedInput
+import { DumbField } from '../Form/fields'
```

Comment:
`Dumb` usually for a component that is not connected to redux, but… we don't really have redux here. Maybe a name like `Input` or `FieldInput` is better?

Reply:
C'est corrigé :)